api/py/test/sample/group_bys/sample_team/sample_group_by_for_dependency_test.py

```diff
@@ -0,0 +1,28 @@
+#     Copyright (C) 2023 The Chronon Authors.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+from ai.chronon.group_by import Aggregation, GroupBy, Operation
+from sources import test_sources
+
+v1 = GroupBy(
```

Comment:
Suggest calling this `sample_simple_group_by` ? Same for join. There might be future test cases that need a simple GB<>Join. Promote reuse instead of creating new